plugins/meta/tuning/tuning.go

```diff
@@ -224,6 +239,9 @@ func createBackup(ifName, containerID, backupPath string, tuningConf *TuningConf
 		config.Allmulti = new(bool)
 		*config.Allmulti = (link.Attrs().RawFlags&unix.IFF_ALLMULTI != 0)
 	}
+	if tuningConf.TxQLen != 0 {
```

Comment:
nit: this is redundant, right ?
`0` is the default value of the `configToRestore` struct; I would always set it, which would allow you to remove this if.

Reply:
I had  to use an int pointer to make sure the tx queue len was being set and unset properly